cmd/ef_tests/test_runner.rs

```diff
@@ -33,7 +28,7 @@ fn execute_test(test: &TestUnit) {
             .clone()
             .unwrap()
             .into(),
-        &pre,
+        &mut build_evm_state_from_prestate(&test.pre),
```

Comment:
this will allow us to execute multiple transactions and blocks, right?

Reply:
Yes